cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/pipeline/CreateProgramSchedulesStage.java

```diff
@@ -63,9 +63,7 @@ public void process(final ApplicationWithPrograms input) throws Exception {
       Trigger trigger = scheduleCreationSpec.getTrigger();
       ProgramSchedule programSchedule =
         new ProgramSchedule(scheduleCreationSpec.getName(), scheduleCreationSpec.getDescription(),
-                            programId,
-                            scheduleCreationSpec.getProperties(), trigger,
-                            new ArrayList<Constraint>());
+                            programId, scheduleCreationSpec.getProperties(), trigger, new ArrayList<Constraint>());
```

Comment:
Should this call ScheduleCreationSpec.getConstraints()? Even if constraints are not implemented yet, once they are, this will have to change.